client/src/tests/merge.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { merge } from "../merge";
+
+const cases: Array<{
+  title: string;
+  left: any;
+  right: any;
```

Comment:
nit: consider `base` and `diff` or `patch` may be clearer

Reply:
yeah you're right

Reply:
done